CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Features
 
 - Added iterative selection on edges.
+- Added pathfinding and its preview for faces
```

Comment:
This should be descriptive enough that a user could understand the effect of the change. Sometimes this means adding context to the changelog entry. Consider something like "Added a selection preview when using the Select Path tool" or similar.